trendmicrovisionone_connector.py

```diff
@@ -835,850 +733,1438 @@ def _handle_remove_from_blocklist(self, param):
+        """
+        self.save_progress(
+            "In action handler for: {0}".format(self.get_action_identifier())
+        )
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Required Params
+        block_objects: List[Dict[str, str]] = json.loads(param.get("block_objects"))
+
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        multi_resp: List[MsData] = []
+
+        # Choose Enum
+        for i in block_objects:
+            obj_type = i["object_type"].upper()
+            if obj_type in ObjectType.__members__:
+                i["object_type"] = ObjectType[obj_type]
+            else:
+                raise RuntimeError(f"Please check object type: {i['object_type']}")
+
+        # Make rest call
+        for i in block_objects:
+            response = client.remove_from_suspicious_list(
+                pytmv1.ObjectTask(
+                    object_type=i["object_type"], object_value=i["object_value"]  # type: ignore
+                )
+            )
+            if "error" in response.result_code.lower():
+                raise RuntimeError(
+                    f"Error while removing from suspicious list: {response.errors}"
+                )
+            assert response.response is not None
+            items = response.response.dict().get("items", [])[0]
+            items["task_id"] = (
+                "None" if items.get("task_id") is None else items["task_id"]
+            )
+            multi_resp.append(items)
+
+        # Get suspicious list count
+        total_suspicious_count = self.suspicious_list_count()
+        # Add the response into the data section
+        action_result.add_data(
+            {"multi_response": multi_resp, "total_count": total_suspicious_count}
+        )
+
+        # Return success
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_check_analysis_status(self, param):
+        """
+        Checks the submission status for item(s) sent to sandbox for analysis.
+        Args:
+            submit_id: The Output ID  generated from Start-Analysis Command.
+        Returns:
+            Dict: Object containing response regarding submission status.
+        """
+        self.save_progress(
+            "In action handler for: {0}".format(self.get_action_identifier())
+        )
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Required Params
+        task_id = param["task_id"]
+
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        # Make rest call
+        response = client.get_sandbox_submission_status(submit_id=task_id)
+        if "error" in response.result_code.lower():
```

Comment:
This could either be a helper function or even a method within pytmv1 along the lines of 

```python
class Response:
    result_code: str
    def is_error(self) -> str:
        return "error" in self.result_code.lower()
```

Reply:
error response is implemented.